ietf-scitt-charter.md

```diff
@@ -1,35 +1,42 @@
+
+[//]: # (MW: The previous introduction sounded like this is a new problem -- it isn't. It's only recently gained importance and recognition due to the increase in interconnected devices. If we are going to cite a rationale to dealing with this now it is the complex supply chain from various commonly used and complexity of component distribution and their diverse sources.)
 
 The output of the SCITT WG is a set of standards that define the essential building blocks enabling the security of supply chain systems and assisting implementers in securing deployments.
 For example, a public computer interface system could report its software composition, which can be compared against known software compositions for such a device, as recorded in a public append-only transparent registry.
 Therefore, providing an individual using the system with confidence that it will behave as and when expected, consistently and without deviation.
 
+[//]: # (MW: "recorded in a public ledger, " I thought that is too implementation specific for an introduction)
+[//]: # (MW: Does "nowhen" belong here?)
+
 Problem Statement
 =================
 It is challenging to manage the compliance of products across end-to-end, global supply chains. 
 
 Some of the fundamental security issues that face the supply chain ecosystem today are as follows:
 
-1. A single product is composed of multiple sub-products coming from different suppliers. There is no agreed-upon standard to compose information from different producers as every vendor is doing it differently.
+1. A single product is composed of multiple sub-products coming from different suppliers. There are several standards to compose supply chain information with different producers choosing different methods.
 2. There are no APIs defined for automated publishing, retrieval, or independent verification of the information above.
-3. The absence of decentralized, globally interoperable, transparent services to publish the supply chain data.
-4. The lack of sufficient standards for independently verifying the presence of supply chain data in tamper-proof data stores.
-5. Software consumers have no trustworthy way to verify that a software signature on a software package is legitimate.
+4. The absence of decentralized, globally interoperable, transparent services to publish the supply chain data.
+5. The lack of sufficient standards for independently verifying the presence of supply chain data in tamper-proof data stores.
+6. Software consumers have no trustworthy way to verify that a software signature on a software package is legitimate.
 
-A minimal, simple, and concise set of building blocks that interact in a standardized way will assure long-term accountability and interoperability for supply chain components throughout their lifecycles across architecturally diverse systems.
+[//]: # (MW: #6 is not true, SWID tags can be signed today.)
```

Comment:
I interpret this as know standard way to discover if they key used to sign the tag is authoritative for the entity in question. SPDX and Cyclone also support signing.